internal/initializer/crds.go

```diff
@@ -131,6 +135,39 @@ func (c *CoreCRDs) Run(ctx context.Context, kube client.Client) error {
 		if err := pa.Apply(ctx, crd); err != nil {
 			return errors.Wrap(err, "cannot apply crd")
 		}
+
+		pendingCRDs[crd.Name] = struct{}{}
 	}
+
+	// wait for crds to be established
+	pollInterval := 2 * time.Second
+	timeout := 2 * time.Minute
+	if err := wait.PollUntilContextTimeout(ctx, pollInterval, timeout, true, func(ctx context.Context) (bool, error) {
+		for crdName := range pendingCRDs {
+			crd := &extv1.CustomResourceDefinition{}
+			err := kube.Get(ctx, client.ObjectKey{Name: crdName}, crd)
```

Comment:
Hey folks @sttts @phisco, I came across NewAPIPatchingApplicator(kube).apply() which is getting used to create/patch the CRDs. I see we're also using namespace during Get, I think it needs to be added here as well then?

Reply:
CRD don't have a namespace.